### PR TITLE
Use `s1_orbits`

### PR DESCRIPTION
**Pull request by jtherrmann**
TODO:

- [x] Don't install `s1_orbits` via `pip` in `environment.yml` after it's published to conda-forge
- [x] Test locally with before and after versions of the plugin and confirm the same orbit files are downloaded.
- [ ] Squash merge to clean up the fact that I manually reverted changes across several files, which will make `git blame` ugly

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.0]
+### Changed
+- Orbit files are now retrieved using the [s1-orbits](https://github.com/ASFHyP3/sentinel1-orbits-py) library.
```
Comment (jtherrmann) [inline CHANGELOG.md:11]:
```suggestion
- Orbit files are now retrieved using the [s1-orbits](https://github.com/ASFHyP3/sentinel1-orbits-py) library.
- Creating a Conda environment from `environment.yml` now installs all dependencies via `pip`, to reduce duplication via `environment.yml` and `pyproject.toml`.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Creating a Conda environment from `environment.yml` now installs all dependencies via `pip`, to reduce duplication via `environment.yml` and `pyproject.toml`.
```
Comment (jtherrmann) [inline CHANGELOG.md:12]:
```suggestion
- Creating a Conda environment from `environment.yml` now installs all dependencies via `pip`, to reduce duplication between `environment.yml` and `pyproject.toml`.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.0]
+### Changed
+- Orbit files are now retrieved using the [s1-orbits](https://github.com/ASFHyP3/sentinel1-orbits-py) library.
+- Creating a Conda environment from `environment.yml` now installs all dependencies via `pip`, to reduce duplication between `environment.yml` and `pyproject.toml`.
```
Comment (jtherrmann) [inline CHANGELOG.md:12]:
```suggestion
```

File: environment.yml [hunk 4]
```diff
@@ -28,3 +28,5 @@ dependencies:
   - pytest-mock
   # For running
   - hyp3lib>=3,<4
+  - pip: # TODO: delete this line after s1_orbits published to conda-forge
+    - s1_orbits
```
Comment (jtherrmann) [inline environment.yml:32]:
```suggestion
  - s1_orbits
```

**Comment by jtherrmann** [comment]
Verified that `python -m hyp3_isce2 ++process insar_tops_burst S1_136231_IW2_20200604T022312_VV_7C85-BURST S1_136231_IW2_20200616T022313_VV_5D11-BURST --looks 20x4 --apply-water-mask True` downloads the same orbit files (same filenames and contents) on `develop` and `s1-orbits`.

**Review by AndrewPlayer3 — APPROVED** [review]

**Pull request merged**